docs/getting-started-e2e.md

```diff
@@ -171,15 +171,15 @@ git branch -u upstream/main main
 
 # Building AMP and starting a local server
 
-Now that you have all of the files copied locally you can actually build the code and run a local server to try things out. We use Node.js, the NPM package manager, and Closure Compiler to build amphtml and start up a local server that lets you try out your changes.
+Now that you have all of the files copied locally you can actually build the code and run a local server to try things out. We use Node.js, the npm package manager, and Closure Compiler to build amphtml and start up a local server that lets you try out your changes.
 
 -   Install the latest LTS version of [Node.js](https://nodejs.org/) (which includes npm). If you're on Mac or Linux, an easy way to install Node.js is with `nvm`: [here](https://github.com/creationix/nvm).
 
     ```sh
     nvm install --lts
     ```
 
--   Closure Compiler is automatically installed by NPM, but it requires Java 8 which you need to install separately. AMP's version of Closure Compiler won't run with newer versions of Java. Download an installer for Mac, Linux or Windows [here](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+-   Closure Compiler is automatically installed by npm, but it requires Java 8 which you need to install separately. AMP's version of Closure Compiler won't run with newer versions of Java. Download an installer for Mac, Linux or Windows [here](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
```

Comment:
This isn't true anymore. We could remove this from the instructions since we use a native binary for Closure Compilation.

Reply:
Done in #34582.